src/reducer/database/models.ts

```diff
@@ -0,0 +1,1066 @@
+  public objects = Organization.objects;
+
+  public id = 0;
+  public avatarUrl = "";
+  public url = "";
+  public name = "";
+  public fullname = "";
+  public description = "";
+  public platformName = "";
+
+  constructor(args: IOrganization) {
+    super();
+
+    this.id = args["id"];
+    this.avatarUrl = args["avatarUrl"];
+    this.url = args["url"];
+    this.name = args["name"];
+    this.fullname = args["fullname"];
+    this.description = args["description"];
+    this.platformName = args["platformName"];
+  }
+
+  /**
+   * Create a member within this organization.
+   *
+   * @param fields Member data
+   * @returns {Member} A member object
+   */
+  createMember(fields: any): Member {
+    let member = Member.objects.create({
+      avatarUrl: fields.avatarUrl,
+      url: fields.url,
+      fullname: fields.fullname,
+      username: fields.username,
+      platformId: fields.platformId,
+    });
+
+    if (member.success === false) {
+      member = Member.objects.filter({
+        url: fields.url,
+      })[0];
+    }
+
+    OrganizationHasMember.objects.create({
+      organizationId: this.id,
+      memberId: member.id,
+    });
+
+    return member as Member;
+  }
+
+  /**
+   * Get all members within this organization.
+   *
+   * @returns {Member[]} A list of member objects
+   */
+  getMembers(): Member[] {
+    let members = OrganizationHasMember.objects.filter(
+      {
+        organizationId: this.id,
+      },
+      Member
+    );
+
+    return members as Member[];
+  }
+
+  /**
+   * Get all repositories which belongs to this organization.
+   *
+   * @returns {Repository[]} A list of repository objects
+   */
+  getRepositories(): Repository[] {
+    let filterStatement = Repository.statements.withOwner;
+    let response = this.objects.filter(
+      { owner: this.name },
+      Repository,
+      filterStatement
+    );
+
+    return response as Repository[];
+  }
+
+  /**
+   * @todo Implement a save functionality
+   * @description Saves this osm model
+   */
+  save() {}
+}
+
+/**
+ * @class A OSM model for a organizationHasMember object
+ * @extends osm.models.OrganizationHasMemberSO Statement object
+ */
+class OrganizationHasMember extends osm.models.OrganizationHasMemberSO {
+  public static objects = osm.models.OrganizationHasMemberSO.getObjects(
+    OrganizationHasMember
+  );
+  public objects = OrganizationHasMember.objects;
+  public id = 0;
+
+  /**
+   * @todo Implement a save functionality
+   * @description Saves this osm model
+   */
+  save() {}
+}
+
+/**
+ * @class A OSM model for a organizationHasMember object
+ * @extends osm.models.PlatformHasOrganizationSO Statement object
+ */
+class PlatformHasOrganization extends osm.models.PlatformHasOrganizationSO {
+  public static objects = osm.models.PlatformHasOrganizationSO.getObjects(
+    PlatformHasOrganization
+  );
+
+  public objects = PlatformHasOrganization.objects;
+  public id = 0;
```

Comment:
A newline has to be added, because this is a logical break and the other classes also had a newline between the id and objects.
```suggestion
  public objects = PlatformHasOrganization.objects;
  
  public id = 0;
```